content/en/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions.md

```diff
@@ -878,19 +755,13 @@ Default values for `metadata` fields of `x-kubernetes-embedded-resources: true`
 
 ### Publish Validation Schema in OpenAPI v2
 
-{{< feature-state state="stable" for_k8s_version="v1.16" >}}
-
-{{< note >}}
-OpenAPI v2 Publishing is available as beta since 1.15, and as alpha since 1.14. The
-`CustomResourcePublishOpenAPI` feature must be enabled, which is the case automatically for many clusters for beta features. Please refer to the [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) documentation for more information.
```

Comment:
the feature gate lifecycle is different from an API deprecation period. I'm not sure what's the convention for feature gates, but I assume we should keep this to let users know a feature gate exists which could enable/disable this feature

Reply:
AIUI you _can't_ disable stable features; that's what makes them stable. If that's wrong then there's a different page we need to fix, I think.

Reply:
This paragraph, IMHO, can be safely removed. 1) 1.15 is a version too old to carry on for docs; 2) the feature gate is already graduated, so it will be ignored.